packages/client/extract-translations.sh

```diff
@@ -8,7 +8,42 @@
 # Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
 # graphic logo are (registered/a) trademark(s) of Plan International.
 
-echo 'The Environment variable COUNTRY_CONFIG_PATH must be set in your Terminal, so we can check that your country configuration has all necessary translations.  If you cd into your country configuration repo and run the command pwd, then this will display for you.  Then run export COUNTRY_CONFIG_PATH=<your country config path> in this window and try to commit again please..  Currently:'
-echo $COUNTRY_CONFIG_PATH
+get_abs_filename() {
+  echo "$(cd "$(dirname "$1")" && pwd)/$(basename "$1")"
+}
+
+WRITE=false
+
+for i in "$@"; do
+  case $i in
+    --write=*)
+      WRITE=true
+      shift
+      ;;
+    -*|--*)
+      echo "Unknown option $i"
+      exit 1
+      ;;
+    *)
+      ;;
+  esac
+done
+
+if [ -z "$COUNTRY_CONFIG_PATH" ] ; then
+  echo 'The Environment variable COUNTRY_CONFIG_PATH must be set in your Terminal, '
+  echo 'so we can check that your country configuration has all necessary translations.'
+  echo 'If you cd into your country configuration repo and run the command pwd, then this will display for you.'
+  echo 'Then run export COUNTRY_CONFIG_PATH=<your country config path> in this window and try to commit again please..'
+  exit 1
```

Comment:
What do you think if we would default this to `../opencrvs-farajaland` if the directory exists?

Reply:
That sounds like a good idea!

Reply:
Not bad! I'll make the change. One idea I also had was to get the localisations through the country config endpoint. It would've worked with the previous complementation but the `--write` flag makes it harder